RoadMap_SwiftUI/RoadMap_SwiftUI/Model/Notification.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  Notification.swift
+//  RoadMap_SwiftUI
+//
+//  Created by Артур Щукин on 27.01.2023.
+//
```

Comment:
```suggestion
//
//  Notification.swift
//  RoadMap_SwiftUI
//
//  Created by Артур Щукин on 27.01.2023.
//
```
не отработал форматер